src/util/common.ts

```diff
@@ -31,3 +31,24 @@ export function createNewPlatform(): Platform {
 
   return platform;
 }
+
+export function updatePoolDayData(geyser: Geyser, timestamp: number): PoolDayData {
```

Comment:
can we also hit this function from the periodic pricing update? that'll make sure we populate the snapshot stats every day

Reply:
good idea